### PR TITLE
OTG Tests Update - Corrected loss percentage calculation

### LINKED ISSUE #560: OTG: RT-1.2: BGP Policy & Route Installation
- [README](https://github.com/openconfig/featureprofiles/blob/main/feature/bgp/policybase/otg_tests/route_installation_test/README.md)
- Initial PR #240 
- Update1 #432

### LINKED ISSUE #568: OTG: TE-3.1: Base Hierarchical Route Installation
- [README](https://github.com/openconfig/featureprofiles/blob/main/feature/gribi/otg_tests/base_hierarchical_route_installation_test/README.md)
- Init PR #348

### LINKED ISSUE #570: OTG: TE-3.5: Ordering: ACK Received
- [README](https://github.com/openconfig/featureprofiles/blob/main/feature/gribi/otg_tests/ordering_ack_test/README.md)
- Init PR #367

### LINKED ISSUE #571: OTG: TE-3.6: ACK in the Presence of Other Routes
- [README](https://github.com/openconfig/featureprofiles/blob/main/feature/gribi/otg_tests/ack_in_presence_other_routes/README.md)
- Init PR #421 
- Update 1 PR #552

### LINKED ISSUE #572: OTG: TE-4.1: Base Leader Election
- [README]()
- Init PR #291

### LINKED ISSUE #573: OTG: TE-4.2: Persistence Mode
- [README](https://github.com/openconfig/featureprofiles/blob/main/feature/gribi/otg_tests/leader_failover_test/README.md)
- Init PR #422

### LINKED ISSUE #575: OTG: TE-6.1: Route Removal via Flush
- [README](https://github.com/openconfig/featureprofiles/blob/main/feature/gribi/otg_tests/route_removal_via_flush_test/README.md)
- Init PR #365

### LINKED ISSUE #589: OTG: RT-2.1: Base IS-IS Process and Adjacencies

### LINKED ISSUE #633: OTG: RT-3.2: Multiple <Protocol, DSCP> Rules for VRF Selection

### LINKED ISSUE #743: OTG: TE-6.2: Route Removal In Non Default VRF

### LINKED ISSUE #894: OTG: TE-8.1: DUT Daemon Failure

### LINKED ISSUE #1005: OTG: TE-11.3: Backup NHG: Actions

### LINKED ISSUE #1329: OTG: TE-13.1: gRIBI route ADD during Failover

### LINKED ISSUE #1445: OTG: P4RT-1.2: P4RT Daemon Failure

### LINKED ISSUE #1878: OTG: OC-1.2: Default Address Families

### PR DESCRIPTION
**Pull request by octpetre**
txPackets = 0 check and float32 loss Pct calculation for some otg tests
Fixes #560 
Fixes #1005 
Fixes #894 
Fixes #589 
Fixes #1445 
Fixes #633 
Fixes #571 
Fixes #568 
Fixes #570 
Fixes #572 
Fixes #743 
Fixes #573 
Fixes #575 
Fixes #1878 
Fixes #1329

### CODE REVIEW

File: internal/otgutils/print.go [hunk 1]
```diff
@@ -30,6 +31,7 @@ import (
 // LogFlowMetrics displays the otg flow statistics.
 func LogFlowMetrics(t testing.TB, otg *otg.OTG, c gosnappi.Config) {
 	t.Helper()
+	time.Sleep(1 * time.Second)
```
Comment (jasdeep-hundal) [inline internal/otgutils/print.go:34]: Assuming there's a good reason for the sleep, can we add a comment here? (Or alternatively, is there a way to add a reasonable Await?)

**Review by jasdeep-hundal — COMMENTED** [review]
Can we actually switch to using loss percentage directly in some of these cases?

File: internal/otgutils/print.go (hunk 1 above)
Comment (octpetre) [reply internal/otgutils/print.go:35]: Basically before every loss Pct calculation we're printing the stats using these otgutils functions and we want this short delay in order to make sure there are no packets in transit while polling. Added comments

**Comment by jasdeep-hundal** [comment]
@octpetre : Needs a more recent merge with main so we can get this in.

**Comment by octpetre** [comment]
> Can we actually switch to using loss percentage directly in some of these cases?

In otg the loss percentage is not available as a stat

**Pull request merged**